### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by nerdinand**

### CODE REVIEW

**Review by crackofdusk — APPROVED** [review]
I usually don't constrain the dependency versions in applications (the lockfile + version resolution should be enough), but I don't have a strong opinion.

File: Gemfile [hunk 1]
```diff
@@ -9,11 +9,11 @@ ruby RUBY_VERSION
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem 'jekyll'
+gem 'jekyll', '~> 4.3', '>= 4.3.2'
```
Comment (lxxxvi) [inline Gemfile:12]: What's the reason for locking the version? Isn't it likely that we want to change again?

**Review by lxxxvi — APPROVED** [review]

File: Gemfile [hunk 2]
```diff
@@ -9,11 +9,11 @@ ruby RUBY_VERSION
...
 
 # Ruby 3: We can remove this dependency after jekyll added 'webrick' to their dependencies
 # => https://github.com/jekyll/jekyll/issues/8523
-gem 'webrick'
+gem 'webrick', '~> 1.8', '>= 1.8.1'
```
Comment (hharen) [inline Gemfile:16]:
```suggestion
gem 'webrick', '~> 1.8', '>= 1.9.1'
```

**Comment by lxxxvi** [comment]
Superseded by #120

**Pull request closed without merging**